R/PredictionSurv.R

```diff
@@ -13,12 +13,22 @@
 #' learner = lrn("surv.kaplan")
 #' p = learner$train(task, row_ids = 1:20)$predict(task, row_ids = 21:30)
 #' head(as.data.table(p))
+#' class(p$data$distr) # survival matrix stored
+#' p$distr # Matdist
 PredictionSurv = R6Class("PredictionSurv",
   inherit = Prediction,
   public = list(
     #' @description
     #' Creates a new instance of this [R6][R6::R6Class] class.
     #'
+    #' @details
+    #' Upon initialization, the `distr` input will be coerced to a survival matrix
+    #' or array (accessible via `$data$distr`) if it's a [Distribution][distr6::Distribution]
+    #' object. The active field `$distr` always returns a distribution
+    #' ([Matdist][distr6::Matdist] or [Arrdist][distr6::Arrdist]) depening on
+    #' the class of the stored `$data$distr`. In the case of an [Arrdist][distr6::Arrdist],
+    #' the distribution is by default initialized using `which.curve = 'mean'`.
+    #'
```

Comment:
See above comment, not always true. Happy for this to be kept in if there's something added about potential for it to always just be a `Distribution`

Reply:
I will rephrase it, I see your point